roles/edpm_network_config/defaults/main.yml

```diff
@@ -40,5 +40,6 @@ edpm_network_config_os_net_config_mappings: {}
 edpm_network_config_safe_defaults: true
 edpm_network_config_with_ansible: true
 edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
+edpm_network_config_nmstate_template: templates/system_role_nmstate/single_nic_vlans.j2
```

Comment:
Rather than making this a file path, which is only really useful for CI can we do a similar thing to this PR?
https://github.com/openstack-k8s-operators/edpm-ansible/pull/243

So instead of having a template file, we can in-line the yaml which will work better for the dataplane-operator CRD interface?

Like this:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/243/files#diff-89da6e02d8fbed04de229b13205144afd3d4bcedda05167972c6badac58102b1R21

Which means the Dataplane operator CRD interface would look like:
https://github.com/openstack-k8s-operators/dataplane-operator/pull/318/files#diff-7c43891912d609cdc0e86a9cb67c44e43264d6e6463d9afa2726a1292cf749caR57-R90

Since users can't realistically use a filepath template file.

Reply:
I should have read the conversation history before my comment. I see this was discussed above too.

Reply:
I modified the role following your PRs examples. I also drop `edpm_network_config_nmstate_template` (re)using `edpm_network_config_template` var because otherwise we should handle this case here: https://github.com/openstack-k8s-operators/dataplane-operator/pull/284/files#diff-874aae70e738f15dc1bf30750cee0e87e3278c5c02d37e35c0943ca6c29393bb